src/components/Toaster.less

```diff
@@ -0,0 +1,26 @@
+@import '~@gen3/ui-component/dist/css/base.less';
```

Comment:
Remove this

Reply:
also rename this `Toaster.less` file into `Toaster.css` since we are not using any less syntax here